pages/form-builder/support/[[...supportType]].tsx

```diff
@@ -1,7 +1,7 @@
 import { serverSideTranslations } from "next-i18next/serverSideTranslations";
```

Comment:
The form structure changed between the two forms. IMO separating the forms like the below makes the most sense vs. a lot of if-statements  in a single form that can be hard to follow. This also takes the forms one step closer to becoming separate pages/files that would be easier to maintain IMO.